README.md

```diff
@@ -1,57 +1,9 @@
-# Northcoders News API
+# Setup Instructions
 
-## Background
-
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
-
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
-
-## Kanban
-
-### Link to your Trello Board here: https://trello.com/b/7yiHe1nI
-
-To keep track of the tasks involved in this project we're going to use a kanban board. Ensure that you work on one _ticket_ at time. You can click on the ticket to find out more information about what is required for the feature. A ticket is not considered complete unless both the happy path and errors response are handled. You can make use of the checklist on each ticket to keep track of the errors you want to handle. You can also make use of [error-handling.md](error-handling.md) to consider the error codes we may wish to respond with.
-
-**Please ensure you work through the tickets in numerical order.**
-
-## Git Branching and Pull Requests
-
-You will be working on each ticket on a new **branch**.
-
-To create and switch to a new git branch use the command:
+Create `.env.development` (and `.env.test` for testing) to add your environment variables for connecting to the database:
 
 ```
-git checkout -b <new branch name>
-```
-
-This will create a branch and move over to that branch. (Omit the `-b` flag if you wish to switch to an already existing branch).
-
-We recommend that you name the branch after the number assigned to each ticket via the header. eg. `ncnews-1`
-
-When pushing the branch to git hub ensure that you make reference to the branch you are pushing to on the remote.
-
+PGDATABASE=database_name_here
+---> you can find database_name in sql file
```

Comment:
Id want this to be a bit more obvious about how to set it up. What do i put in each file? How do i know which pgdatabase to put in the right file? Assume that the person reading this has never seen your repo before and does not know how it works. Give more detailed instructions